submittable/component-library
# SUB-6204 feat(commitizen): add commitizen for semantic commit convention

---
[SUB-6204](https://submittable.atlassian.net/browse/SUB-6204) add commitizen for semantic commit conventions
---
### Description
We should standardize how we commit to the code base and structure it in a way where it can simply help identify the version the package should be publish base on the commit history. 

Read and understand why follow this convention
[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) 
How to enforce commits into the repo
[GitHub - commitizen/cz-cli: The commitizen command line utility. #BlackLivesMatter](https://github.com/commitizen/cz-cli) 


### Acceptance Criteria
Have commitizen setup so when a developer were to make a commit to the repo it would follow semantic commit convention

### Demo

https://user-images.githubusercontent.com/97060116/179628436-f99ce975-a5b0-41ed-837a-ecd0a51b8053.mov



### Testing
1. pull down branch npm install
2. make a change to a file (add a space to the README)
3.  `git add .`
4. `git commit`
5. Confirm Commitizen is working (just :wq to exit if you want) :)
